conversions/Forwards/src/main/java/com/agentdid127/resourcepack/forwards/impl/PackMetaConverter.java

```diff
@@ -102,7 +102,7 @@ else if (to >= Util.getVersionProtocol(packConverter.getGson(), "1.7.2")
                 && to >= Util.getVersionProtocol(packConverter.getGson(), "1.20.2")) {
                 JsonObject supportedFormats = new JsonObject();
                 supportedFormats.addProperty("min_inclusive", versionInt);
-                supportedFormats.addProperty("max_inclusive", Util.getLatestProtocol(packConverter.getGson()));
+                supportedFormats.addProperty("max_inclusive", 26); // TODO: fix hardcoded bruh
```

Comment:
can we do versionInt for now until we can decide on a better way to do this?

Reply:
fine